conans/test/integration/command/info/test_info_folders.py

```diff
@@ -0,0 +1,150 @@
+import os
+import textwrap
+import json
+import pytest
+
+from conans.paths import CONANFILE
+from conans.test.utils.tools import TestClient
+from conan.internal.cache.conan_reference_layout import EXPORT_FOLDER
+
+conanfile_py = """
+from conan import ConanFile
+
+class AConan(ConanFile):
+    name = "package"
+    version = "0.1.0"
+    short_paths=False
+"""
+
+with_deps_path_file = """
+from conan import ConanFile
+
+class BConan(ConanFile):
+    name = "package2"
+    version = "0.2.0"
+    requires = "package/0.1.0@user/testing"
+"""
+
+deps_txt_file = """
+[requires]
+package2/0.2.0@user/testing
+"""
+
+user = "user"
+channel = "testing"
+user_channel = f"{user}/{channel}"
+reference1 = f"package/0.1.0@{user_channel}"
+reference2 = f"package2/0.2.0@{user_channel}"
```

Comment:
But why have globals? They can be safely defined inside the tests, it is more clear

Reply:
Done, removed